### PR TITLE
added TryGetController method

### PR DESCRIPTION
**Pull request by StephenHodgson**
Overview
---
Added `TryGetController` utility method to Input System

Changes
---
- Fixes: #3299

### CODE REVIEW

File: Assets/MixedRealityToolkit-SDK/Features/Input/MixedRealityInputManager.cs [hunk 1]
```diff
@@ -470,6 +470,24 @@ public void ClearFallbackInputStack()
 
         #endregion Fallback Input Handler Options
 
+        #region IMixedRealityController Utilities
+
+        /// <inheritdoc />
+        public IMixedRealityController TryGetController(IMixedRealityInputSource inputSource)
```
Comment (keveleigh) [inline Assets/MixedRealityToolkit-SDK/Features/Input/MixedRealityInputManager.cs]: Should this be `public bool TryGetController(IMixedRealityInputSource inputSource, out IMixedRealityController controller)` to match some of our other `TryGet` methods? I'm not sure if we've been consistent with the naming/behavior everywhere, but all the ones I just found on my search behave this way.

Comment (StephenHodgson) [reply Assets/MixedRealityToolkit-SDK/Features/Input/MixedRealityInputManager.cs]: Sure I can update it to be that instead :)

File: Assets/MixedRealityToolkit/_Core/EventDatum/Input/InputEventData.cs [hunk 2]
```diff
@@ -16,7 +17,14 @@ public class InputEventData : BaseInputEventData
         /// <summary>
         /// Handedness of the <see cref="IMixedRealityInputSource"/>.
         /// </summary>
-        public Handedness Handedness { get; private set; } = Handedness.None;
+        public Handedness Handedness
+        {
+            get
+            {
+                var controller = MixedRealityToolkit.InputSystem.TryGetController(InputSource);
```
Comment (StephenHodgson) [inline Assets/MixedRealityToolkit/_Core/EventDatum/Input/InputEventData.cs]: I suppose with this change we should also remove the handedness property in `Initialize`

Comment (StephenHodgson) [reply Assets/MixedRealityToolkit/_Core/EventDatum/Input/InputEventData.cs]: Or we can remove `eventData.Handedness` all together

Comment (keveleigh) [reply Assets/MixedRealityToolkit/_Core/EventDatum/Input/InputEventData.cs]: Strongly against removing `eventData.Handedness`. It provides easy, discoverable access to data that is valuable when performing handed interactions.

Comment (StephenHodgson) [reply Assets/MixedRealityToolkit/_Core/EventDatum/Input/InputEventData.cs]: Pointers shouldn't know about hands so it wouldn't matter.

**Pull request closed without merging**